### ISSUE TITLE
cannot find element on iOS

### ISSUE THREAD

**Issue by sagar-patil-mmt** [facebookarchive/WebDriverAgent#1122]
Element is visible and present on the screen but Appium still can't find it. I tried accessing the element using accessibility id and Xpath but it doesn't work. I am using Appium 1.10.0, Xcode - 10.1, iOS - 12.1.3.
I get following response from Appium

_[Xcode] 
[JSONWP Proxy] Got response with status 200: {"value":[],"sessionId":"4FC0B498-481B-4599-9A60-B8360E70A9C8","status":0}
[XCUITest] Connection to WDA timed out
[iProxy] recv failed: Operation not permitted
[MJSONWP (db841367)] Responding to client with driver.findElements() result: []
[HTTP] <-- POST /wd/hub/session/db841367-2681-4f50-89d4-89da7579f5b6/elements 200 11446 ms - 74_

I think the issue is because of following reason

My screen reloads constantly(with an interval of 1 sec)
If I reload the screen after 3 sec, It works some time( works when tried finding the element with accessibility id and does not work with xpath).

But **Appium Studio finds element even though the screen reloads after every sec and successfully performs the click operation on that element**.

Does anybody know how does Appium Studio find it and Appium does not?

Another issue is when the element is found (after changing the reload interval to 3 Sec), I some time get the following error

_element.click()] Error response status: 13, UnknownError - An unknown server-side error occurred while processing the command. Selenium error: An unknown server-side error occurred while processing the command. Original error: Error Domain=com.facebook.WebDriverAgent Code=1 "The element '"Accessibility_PrimaryZoneCloseButton" Button' is not visible on the screen and thus is not interactable" UserInfo={NSLocalizedDescription=The element '"Accessibility_PrimaryZoneCloseButton" Button' is not visible on the screen and thus is not interactable}_

I added the StaleElementException but it does not throw that exception.
